src/lib/snark_worker/prod.ml

```diff
@@ -56,12 +56,27 @@ module Inputs = struct
     Snark_work_lib.Work.Single.Spec.Stable.Latest.t
   [@@deriving bin_io_unversioned, sexp]
 
-  type zkapp_command_inputs =
-    ( Transaction_witness.Zkapp_command_segment_witness.t
-    * Transaction_snark.Zkapp_command_segment.Basic.t
-    * Transaction_snark.Statement.With_sok.t )
-    list
-  [@@deriving sexp, to_yojson]
+  let zkapp_command_inputs_to_yojson =
+    let convert =
+      List.map
+        ~f:(fun
+             ( (witness : Transaction_witness.Zkapp_command_segment_witness.t)
+             , segment
+             , statement )
+           ->
+          ( Transaction_witness.Zkapp_command_segment_witness
+            .read_all_proofs_from_disk witness
+          , segment
+          , statement ) )
+    in
+    let impl =
+      [%to_yojson:
```

Comment:
Logging format change, a minor one

Used only in fatal logs, so not frequently used, and is mostly for debugging